### PR TITLE
Redirect cookbook

### PR DESCRIPTION
**Pull request by celestehorgan**
# What changed, and why it matters

This PR is is the "mechanical" parts of migrating the Aiven Terraform Cookbook recipes to the developer center. It does the following: 

1. Redirects the URLs for all Terraform Cookbook pages to developer center pages
2. Removes the Terraform Cookbook from toc.yml
3. Removes the files for the cookbook

# Part 1 of 2 

An additional PR will be created to manually correct links to the terraform cookbook that appear elsewhere in the docs, as redirects tend to lower or SEO :) 

# Do not merge yet

Hold on this PR until the pages in Sanity CMS are approved and published. Then, we will test the redirects manually to ensure (hope?) that they don't break. The redirects won't work until the page is published so this build won't pass until that happens anyways.

Submitting this as not-a-draft because I want the build to run (and fail if required).

### CODE REVIEW

File: docs/products/clickhouse/howto/manage-users-roles.rst [hunk 1]
```diff
@@ -141,3 +141,8 @@ Preview users and roles in the console
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can also see the users, their roles, and permissions in the  `Aiven web console <https://console.aiven.io/>`_. Go to your service page, and select **Users and roles** from the sidebar. Next to every user listed, there is a **View details & grants** button, which shows you a list of all grants for that user.
+
+Manage using Terraform
+--------------
```
Comment (staceysalamon-aiven) [inline docs/products/clickhouse/howto/manage-users-roles.rst:146]:
```suggestion
------------------------
```

File: docs/products/clickhouse/howto/manage-users-roles.rst [hunk 2]
```diff
@@ -141,3 +141,8 @@ Preview users and roles in the console
...
+
+You can also manage user roles and access using the `Aiven Terraform Provider</docs/tools/terraform>`. See `The Aiven Terraform Provider Cookbook recipe: Manage user privileges for Aiven for Clickhouse® services using Terraform"<https://aiven.io/developer/manage-user-privileges-clickhouse-terraform>` for more information.
```
Comment (staceysalamon-aiven) [inline docs/products/clickhouse/howto/manage-users-roles.rst:148]:
```suggestion
You can also manage user roles and access using the `Aiven Provider for Terraform </docs/tools/terraform>`. Try the Aiven Terraform Provider Cookbook recipe, Manage user privileges for Aiven for Clickhouse® services using Terraform<https://aiven.io/developer/manage-user-privileges-clickhouse-terraform>`.
```

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Just a couple of very minor changes. Thanks for doing this!

**Comment by celestehorgan** [comment]
So the Build HTML check is still failing, but now it's failing because it can't find certain links. I'm going to publish the files in their new locations and rerun the builds and see if that addresses the issue.

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM

**Pull request merged**